### ISSUE TITLE
Still under development?

### ISSUE THREAD

**Issue by indiejoseph** [matt2000/tnn-js#1]
Great job @matt2000 , this project let me more clear about TNN, is it still under development?

**Comment by matt2000** [comment]
Not really. If I were to find time to work on TNN again, I would probably
re-implement it from scratch. This version is pretty bad.

All the Best,

Matt
